code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -583,12 +583,53 @@
 	item_state = "gold_id"
 	registered_name = "Captain"
 	assignment = "Captain"
+	var/list/locked_access
+	var/fLocked = TRUE
 
 /obj/item/weapon/card/id/captains_spare/New()
 	var/datum/job/captain/J = new/datum/job/captain
-	access = J.get_access()
+	locked_access = J.get_access()
+	access = list(access_maint_tunnels, access_heads)
 	..()
 
+/obj/item/weapon/card/id/captains_spare/proc/toggle_access_lock(mob/user, var/datum/data/record/R, var/own_hand = TRUE)
+	if(!R)
+		to_chat(user, "<span class = 'warning'>\The [src] makes a soft buzzing noise as [own_hand ? "your fingerprint" : "the severed hand's fingerprint"] is rejected</span>")
+		return
+	if(R.fields["rank"] == "Captain" || R.fields["rank"] == "Head of Security" || R.fields["rank"] == "Research Director" || R.fields["rank"] == "Chief Medical Officer" || R.fields["rank"] == "Head of Personnel" || R.fields["rank"] == "Chief Engineer")
```

Comment:
There should be some const or define for these somewhere.

May also want to change this to a `ALLOWED_RANKS.Find(R.fields["rank"])` check, where `ALLOWED_RANKS` is a `list()` of ranks.

Reply:
`R.fields in command_positions`